public_suffix_list.dat

```diff
@@ -11797,7 +11797,7 @@ usercontent.jp
 // Gentlent, Inc. : https://www.gentlent.com
 // Submitted by Tom Klein <tom@gentlent.com>
 gentapps.com
-lab.ms
```

Comment:
Removal of entries, even coupled within a header section like this requires the domains be validated.

We'll need the removal of lab.ms to be submitted with DNS authority as a separate PR

Please revise this back in

Reply:
Change commited